src/styles/blocks/card.scss

```diff
@@ -0,0 +1,120 @@
+$cardWidth: 200px;
```

Comment:
Please move all variables to separate file: _variables.scss, it could be in utils folder